calico_containers/calicoctl.py

```diff
@@ -108,13 +108,15 @@
 from pycalico.datastore import (ETCD_AUTHORITY_ENV,
                                ETCD_AUTHORITY_DEFAULT,
                                Rule,
-                               Rules,
-                               DataStoreError,
-                               ProfileNotInEndpoint,
+                                ProfileNotInEndpoint,
                                ProfileAlreadyInEndpoint,
                                MultipleEndpointsMatch,
                                BGPPeer,
                                IPPool)
+from calico_containers.pycalico.datastore_errors import DataStoreError, \
```

Comment:
Can we use a common allignment style of "from import" at least within the same file.